nengo/networks/workingmemory.py

```diff
@@ -37,12 +35,14 @@ def InputGatedMemory(n_neurons, dimensions, feedback=1.0,
         # gate difference (if gate==0, update stored value,
         # otherwise retain stored value)
         net.gate = nengo.Node(size_in=1)
+        net.diff.add_neuron_input()
         nengo.Connection(net.gate, net.diff.neuron_input,
                          transform=np.ones((n_total_neurons, 1)) * -10,
                          synapse=None)
 
         # reset input (if reset=1, remove all values, and set to 0)
         net.reset = nengo.Node(size_in=1)
+        net.mem.add_neuron_input()
         nengo.Connection(net.reset, net.mem.neuron_input,
```

Comment:
Nit: could demonstrate the alternative of moving the above line `net.mem.add_neuron_input()` to replace `net.mem.neuron_input`.